bedrock/mozorg/templates/mozorg/about/leadership/index.html

```diff
@@ -9,377 +9,11 @@
-
-      <p>Carlos holds an undergraduate degree in History from UCLA and a J.D. from
-        Stanford Law School. As a first-generation college graduate, Carlos
-        believes in the power of education and enjoys supporting education
-        programs and organizations. He lives with his wife and daughter in San
-        Francisco.</p>
-    </div>
-  </article>
-
-  <article id="imo-udom" data-id="imo-udom" class="vcard has-bio" itemscope itemtype="http://schema.org/Person">
-    <figure class="headshot">
-      {{ resp_img(
-        url='img/mozorg/about/leadership/imo-udom.jpg',
-        srcset={
-          'img/mozorg/about/leadership/imo-udom-high-res.jpg': '2x'
-        },
-        optional_attributes={
-          'width': '160',
-          'height': '160',
-          'class': 'photo',
-          'itemprop': 'image'
-        }
-      ) }}
-      <figcaption><h4 class="fn" itemprop="name">Imo Udom</h4></figcaption>
-    </figure>
-
-    <div class="person-info">
-      <p class="title" itemprop="jobTitle">SVP of Innovation Ecosystems</p>
-
-      <ul class="elsewhere">
-        <li><a class="url mastodon" itemprop="url" rel="me external" href="https://mozilla.social/@imo">@imo@mozilla.social</a></li>
-      </ul>
-
-      <ul class="utility">
-        <li><a class="link" href="{{ url('mozorg.about.leadership.index') }}#imo-udom">Link to this bio</a></li>
-      </ul>
-    </div>
-
-    <div class="person-bio">
-      <p>As Mozilla’s Senior Vice President of Innovation Ecosystems, Imo is
-        responsible for cultivating new innovative ideas and technology from
-        conception to launch outside Mozilla’s core products. In addition to product
-        innovation, Imo is also focused on creating systems to foster collaboration
-        across an ecosystem of stakeholders to help realize Mozilla’s vision to
-        build a better internet.</p>
-
-      <p>Previously, Imo was the Chief Strategy and Product Officer at Outmatch
-        where he was responsible for ensuring the business and product strategy
-        delivered value to customers, while never losing sight of its mission to
-        match people with purpose.</p>
-
-      <p>Prior to Outmatch, Imo co-founded and served as CEO of Wepow, where he
-        grew the video interviewing solution from a small side-project in 2010
-        to a successful enterprise platform supporting hundreds of global brands
-        and later through its acquisition by Outmatch.</p>
-
-      <p>Imo enjoys volunteering and giving back to the community. He is on the
-        board of the Delta San Francisco Peninsula Foundation, a non-profit whose
-        mission is to improve the quality of life in the San Francisco and Peninsula
-        area by supporting programs that foster educational growth and to provide
-        financial assistance in the form of scholarships and awards.</p>
-    </div>
-  </article>
-
-</div>
+  {% include "mozorg/about/leadership/includes/executive.html"%}
```

Comment:
Ooo very smart doing these as includes! This could have easily turned into a massive template that would scroll forever.